src/lib/blueprint.ts

```diff
@@ -126,6 +126,7 @@ interface BaseParameter {
   undocumentedMessage: string
   isDraft: boolean
   draftMessage: string
+  default?: any
```

Comment:
Per you comment, this can be moved off of the base type and defined per specific type to match the parameter type. 
```suggestion
```

Reply:
oh ya that works lol

- Moved to specific types in https://github.com/seamapi/blueprint/pull/178/commits/a6d80531e9b9b127dbd7b21ad9fedb3f1efc4896 so consumers get proper types
- Extended `baseParam` with `any` default https://github.com/seamapi/blueprint/pull/178/commits/a6d80531e9b9b127dbd7b21ad9fedb3f1efc4896 because `parsedProp.default` is `any` anyway.